source/mongorestore.txt

```diff
@@ -215,6 +215,23 @@ Starting in MongoDB 5.0, you can use ``mongorestore`` to restore
 :ref:`timeseries collections <manual-timeseries-landing>`.
 For details, see :ref:`mongorestore-example-time-series`.
 
+Using ``mongorestore`` on Atlas Free and Shared Tier Clusters 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On free (``M0``) and shared (``M2`` and ``M5``) tier Atlas clusters, the
+following limitations apply:
+
+- You can't run ``mongorestore`` on the ``admin`` database. If you use
+  the :option:`--db <mongorestore --db>` option to set the destination
+  database to ``admin``, the program returns an error. By default,
+  ``mongorestore`` skips this option.
```

Comment:
```suggestion
  ``mongorestore`` skips this database.
```

Reply:
I would also add this sentence above the previous one.

Also, I think that you should add a similar sentence to the mongodump section